app/graphql/types/pageable.rb

```diff
@@ -16,6 +16,7 @@ def results_type(&block)
 
           field :totalCount, !types.Int, "Count of total results", property: :total_count
           field :nodes, types[wrapped_type], "A list of nodes."
+          field :facets, types[Types::Facets::FieldType], "A list of facets.", property: :facets
```

Comment:
Are we okay with saying all paginatable responses will have a `facets` field knowing that objects that are backed by the db won't have this ability?

Reply:
hmm I guess I could move that field into the block param, or make it conditional based on a param. Do you have a preference?

Reply:
I guess i was thinking making it conditional based on a param or defining two types `results_type` and `results_with_facet_type`. 

I do think this is work that can be ticketed for later tho if you want. I am going to approve the PR.

Reply:
ok, sounds good